04/app.js

```diff
@@ -0,0 +1,43 @@
+
+const divs = document.querySelectorAll('div');
+const body = document.body;
+
+
+divs.forEach(div => {
+    div.addEventListener('click', function (e) {
+        e.stopPropagation(); 
```

Comment:
Nie chcemy zatrzymywać propagacji tylko z niej skorzystać! 
Jeśli zakomentujesz ten element to kod w liniach 25-43 będzie nie potrzebny :)